### PR TITLE
Add snap USE flag for gnome-extra/gnome-software

### PR DESCRIPTION
**Pull request by Szowisz**

### CODE REVIEW

**Comment by AndrewAmmerlaan** [comment]
This needs rebasing and `package.use.mask` entries for arches that are not amd64. Otherwise LGTM.

File: gnome-extra/gnome-software/gnome-software-45.1.ebuild [hunk 1]
```diff
@@ -30,6 +30,10 @@ RDEPEND="
 		>=sys-apps/flatpak-1.14.0-r1
 		dev-util/ostree
 	)
+	snap? (
+		app-containers/snapd
+		sys-libs/snapd-glib
```
Comment (AndrewAmmerlaan) [inline gnome-extra/gnome-software/gnome-software-45.1.ebuild:35]:
```suggestion
		sys-libs/snapd-glib:=
```

**Comment by mattst88** [comment]
Looks reasonable, but I'm not maintaining GNOME anymore. You probably want @leio

**Comment by flint2** [comment]
Port to 46.2 merged https://github.com/gentoo/gentoo/commit/10481620ee0868ef645eabfc8f45e14efd4faef8

**Pull request closed without merging**